orm/src/test/java/org/hibernate/search/test/configuration/IndexEmbeddedProgrammaticallyMappedTest.java

```diff
@@ -48,13 +48,13 @@ public void canSetIncludeEmbeddedObjectIdProgrammatically() throws Exception {
 						.indexEmbedded()
 							.includeEmbeddedObjectId( true );
 
-			setupTestData( builder );
+			Long scotlandCountryId = setupTestData( builder );
 
 			FullTextSession s = builder.openFullTextSession();
 
 			// when
 			QueryParser parser = new QueryParser( "id", TestConstants.standardAnalyzer );
-			org.apache.lucene.search.Query luceneQuery = parser.parse( "country.id:1" );
+			org.apache.lucene.search.Query luceneQuery = parser.parse( "country.id:" + scotlandCountryId );
```

Comment:
Yes, this fix is needed or it fails. The sequence id is no 1 anymore, and I think it might be different depending on the dialect.